core/templates/pages/story-editor-page/story-editor-page.component.ts

```diff
@@ -132,7 +144,7 @@ angular.module('oppia').component('storyEditorPage', {
         ctrl.validationIssues = ctrl.story.validate();
         var nodes = ctrl.story.getStoryContents().getNodes();
         let skillIdsInTopic = StoryEditorStateService.getSkillSummaries().map(
-          skill => skill.id);
+          story => story.id);
```

Comment:
Why did this change?

Reply:
Also, if we return a list of storyIds, should we change the variable name too?

Reply:
Sorry for that. I think this got changed when doing "find and replace". Removed that now.